### PR TITLE
SDKS-3595 Remove Deprecated Sample Apps

### PR DESCRIPTION
**Pull request by witrisna**
# JIRA Ticket

[SDKS-3593](https://pingidentity.atlassian.net/browse/SDKS-3593)

# Description

SDKS-3595 Remove Deprecated Sample Apps

### CODE REVIEW

File: e2e/app/src/main/java/com/example/app/env/EnvViewModel.kt [hunk 1]
```diff
@@ -0,0 +1,213 @@
...
+
+const val USER_PROFILE_JOURNEY = "UserProfile"
+
+class EnvViewModel : ViewModel() {
+
+    val servers = mutableListOf<FROptions>()
+
+    val localhost = FROptionsBuilder.build {
+        server {
+            url = "https://andy.petrov.ca/openam"
```
Comment (spetrov) [inline e2e/app/src/main/java/com/example/app/env/EnvViewModel.kt:33]: Not sure if we should keep these environments in the sample app... 
I think our docs has some references to this file... Let's confirm with @forgerock-chris though...

**Review by spetrov — APPROVED** [review]
LGTM, but let's confirm if we want to keep all the environments in the sample app...

File: e2e/app/src/main/java/com/example/app/env/EnvViewModel.kt (hunk 1 above)
Comment (rodrigoareis) [reply e2e/app/src/main/java/com/example/app/env/EnvViewModel.kt:33]: I agree, we should remove those. Ideally we should load these configs from a file, and ignore this file on commits.

**Review by rodrigoareis — DISMISSED** [review]
Looks good to me.

File: e2e/app/src/main/java/com/example/app/env/EnvViewModel.kt (hunk 1 above)
Comment (witrisna) [reply e2e/app/src/main/java/com/example/app/env/EnvViewModel.kt]: Revert to sample env.

**Review by spetrov — APPROVED** [review]
LGTM! 👍🏻

**Pull request merged**